### PR TITLE
Blog post for packaging announcement.

### LINKED ISSUE #116: Announcement for packaging projects
Let's write a blog post to announce that we are looking for people to help with Apple and Windows packaging.

### PR DESCRIPTION
**Pull request by wetneb**
Very much interested in feedback on this, in particular:
* what do you think of this application procedure? Is it ok to send the proposals (including price) in public or should they be directed to some more private address (like hiring@openrefine.org?)
* is it okay not to announce a total budget for this? Perhaps people will not really know the scale of the effort.
* any other issues that should be in the scope of this effort?
* the github issues are a bit messy - perhaps they should be reformulated to be a bit more specific and well written-up?

Closes #116.

### CODE REVIEW

**Comment by thadguidry** [comment]
I would hope for transparency with budget and spending always.  We don't necessarily have to publicly announce a contractor name or address given the work to do, but it inevitably becomes somewhat public if they do a commit or direct PR.

File: _posts/2022-09-30-windows-macos-packaging.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+* (MacOS) [Sign and notarize our DMG distribution (#2191)](https://github.com/OpenRefine/OpenRefine/issues/2191)
+* (Windows) [Offer a proper installer / uninstaller on Windows (#3224)](https://github.com/OpenRefine/OpenRefine/issues/3224)
+* (Windows) [Give an easier way to start and stop OpenRefine with a system tray integration and log viewer (#3221)](https://github.com/OpenRefine/OpenRefine/issues/3221)
+* (Windows) [Configuration for OpenRefine on Windows should use only 1 config file (.ini) (#3057)](https://github.com/OpenRefine/OpenRefine/issues/3057)
+* (Windows) [Sign openrefine.exe to eliminate extra security warnings (#3003)](https://github.com/OpenRefine/OpenRefine/issues/3003)
+
+All new packaging steps should be integrated in our Continuous Deployment infrastructure (currently running on GitHub Actions), if not in our Maven packaging configuration itself.
+
+We have funding to contract out this work to freelancers.
+To respond to this opportunity, please send the following to our developers mailing list (openrefine-dev@googlegroups.com):
```
Comment (trnstlntk) [inline _posts/2022-09-30-windows-macos-packaging.md:25]: I'm not comfortable with this procedure. It is a large hurdle, and feels like a privacy transgression, to send a quote with price to a mailing list of 222 subscribers, many of whom we don't know at all ourselves. From the subscribers' side, I'm not sure if we should force a series of emails with quotes on them. Our goal should be to find good people, including encouraging promising but perhaps shy/vulnerable/underrepresented people to step up, and this doesn't sound like the best way to do it.

As an alternative, I suggest to ask on our user mailing list (the packaging will benefit end users) if people volunteer to be in a small short-term selection committee, and ask applicants to send applications to this group.

**Comment by trnstlntk** [comment]
I'm not comfortable with the proposed procedure to send applications to the dev mailing list. It is a large hurdle to send a quote with price to a mailing list of 222 subscribers, many of whom we don't know at all ourselves. From the subscribers' side I'm not sure if we should force a series of emails with quotes on them. Our goal should be to find good people, including encouraging promising but perhaps shy/vulnerable/underrepresented people to step up, and this doesn't sound like the best way to do it.

As an alternative, I suggest to ask on our user mailing list (the packaging will benefit end users) if people volunteer to be in a small short-term selection committee, and ask applicants to send applications to this group.

Transparency can be ensured by publishing the spending on these work packages in public (which I also want to do for previous grants). I think our wiki would be a good place for that.

File: _posts/2022-09-30-windows-macos-packaging.md (hunk 1 above)
Comment (trnstlntk) [reply _posts/2022-09-30-windows-macos-packaging.md:25]: I retreat my comment. Let's try it this way.

That said, I feel very responsible for having good packaging solutions for our Windows and MacOS users, and I want the right people to work on it.

If, say, by end November 2022, we haven't found suitable candidates via the above suggested procedure, then I want to step in and recruit candidates with another method.

**Review by trnstlntk — APPROVED** [review]

**Comment by trnstlntk** [comment]
@wetneb please merge the PR as soon as you feel ready to do so. (We should then be prepared to see applications coming in, and responding to them)

**Comment by thadguidry** [comment]
@wetneb It's the weekend coming, let's get the ball rolling to see applications come in...Merge Please  (and I'll ping a bunch of Rust folks with system chops on Windows to help with Windows side of things)

**Review by wetneb — COMMENTED** [review]
I still think it makes sense to direct applications to `hiring@openrefine.org`, only encouraging people to discuss their proposals publicly beforehand (for preliminary feedback).

**Comment by thadguidry** [comment]
@wetneb OK.

**Comment by wetneb** [comment]
Ah, and the hiring@openrefine.org does not seem to exist… I need to fix that before merging.

**Pull request merged**